acceptance/testdata/pack_fixtures/report_output.txt

```diff
@@ -2,7 +2,7 @@ Pack:
   Version:  {{ .Version }}
   OS/Arch:  {{ .OS }}/{{ .Arch }}
 
-Default Lifecycle Version:  0.7.4
+Default Lifecycle Version:  0.7.5
```

Comment:
I think it should be rolled back to 0.7.4.

Reply:
We always try to keep the Default Lifecycle Version as the latest supported/tested released version. In this case it should be 0.7.5 unless there is something I'm unaware of.

Reply:
Does it mean that we update this text file on every new lifecycle version?